bananas_server/web_routes.py

```diff
@@ -20,6 +21,33 @@
 CDN_ACTIVE_URL = []
 
 
+class WebsocketTransport:
+    def __init__(self, ws, source):
+        self._ws = ws
+        self._source = source
+
+    def is_closing(self):
+        return False
+
+    async def write(self, data):
+        await self._ws.send_bytes(data)
+
+    def abort(self):
+        self._ws.do_exit = True
+
+    def close(self):
+        self._ws.do_exit = True
+
+    def get_extra_info(self, what):
+        if what != "peername":
+            raise Exception("Unknown get_extra_info() request")
```

Comment:
raising bare Exception class is a bit bad. Is there a more appropriate specific exception type that could be used?